commands/deploy_action.go

```diff
@@ -63,18 +64,38 @@ func deployAction(c *cli.Context) error {
 		return err
 	}
 
-	if prodString == "" {
-		if groupInfo.Staging.Deployable {
-			prod = 0
-		} else {
-			prod = 1
-		}
+	if staging && prodBool {
+		return cli.NewExitError("`--prod` and `--staging` flags are mutually exclusive", 1)
+	}
+	if staging {
+		prod = 0
+	} else if prodBool {
+		prod = 1
 	} else {
-		prod, err = strconv.Atoi(prodString)
+		logp.Info("`lean deploy` now has no default environment, please add `--prod` or `--staging` flag")
```

Comment:
```suggestion
		logp.Info("`lean deploy` now has no default environment. Specify the target environment by `--prod` or `--staging` flag to avoid this prompt.")
```

会不会有点啰嗦？出发点是觉得「please add」更下面的 prompt 有点矛盾。

Reply:
我觉得不啰嗦。而且「啰嗦」已经成了 Node.js、Rust 等社区很多库推崇的风格。